src/phoenix/server/api/queries.py

```diff
@@ -85,17 +85,25 @@ async def projects(
             last=last,
             before=before if isinstance(before, CursorString) else None,
         )
-        async with info.context.db() as session:
-            projects = await session.scalars(select(models.Project))
-        data = [
-            Project(
-                id_attr=project.id,
-                name=project.name,
-                gradient_start_color=project.gradient_start_color,
-                gradient_end_color=project.gradient_end_color,
+        stmt = (
+            select(models.Project)
+            .outerjoin(
+                models.Experiment,
+                models.Project.name == models.Experiment.project_name,
```

Comment:
I think we should ultimately parameterize this. Otherwise hard to find this project. Non blocking